Gemfile.lock

```diff
@@ -431,7 +434,7 @@ GEM
       activesupport (>= 4.2)
       rack-proxy (>= 0.6.1)
       railties (>= 4.2)
-    websocket-driver (0.7.1)
+    websocket-driver (0.7.2)
```

Comment:
### [websocket-driver](https://github.com/faye/websocket-driver-ruby)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.7.1 → 0.7.2

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **7** commits. See the full changes on [the compare page](https://github.com/faye/websocket-driver-ruby/compare/0.7.1...0.7.2).

These are the individual commits:
- (e568366) [Formatting change: {...} should have spaces inside the brace…](https://github.com/faye/websocket-driver-ruby/commit/e568366f82ffd284208ad7b0081e09041e3354ec)
- (d05eb72) [Mention license change in the changelog](https://github.com/faye/websocket-driver-ruby/commit/d05eb72d2d815d4e1b54a115a8fd4c18606ef74a)
- (b00f587) [Emit ping and pong events from the Server driver](https://github.com/faye/websocket-driver-ruby/commit/b00f58701ec008145d5075075d12271b762381db)
- (202e180) [Allow rack.input.read to return a frozen string](https://github.com/faye/websocket-driver-ruby/commit/202e1800e0e948d83ec18368bdc6cd4651c36eb0)
- (db09a1a) [Update Ruby versions 2.4 to 2.7 on Travis](https://github.com/faye/websocket-driver-ruby/commit/db09a1af222e504ae85e9947cc8170fddedccd7a)
- (46aafa4) [Set {source,target}_version to work with the current version…](https://github.com/faye/websocket-driver-ruby/commit/46aafa4e9963c344775318e7a646fb4f6b9e7d29)
- (f882bc5) [Bump version to 0.7.2](https://github.com/faye/websocket-driver-ruby/commit/f882bc56d530564614c5ad81e61eb99c445bd99a)


</details>